google-cloud-pubsub/lib/google/cloud/pubsub/service.rb

```diff
@@ -255,11 +255,7 @@ def streaming_pull request_enum
         ##
         # Acknowledges receipt of a message.
         def acknowledge subscription, *ack_ids
-          begin
```

Comment:
Why are we removing the try catch as part of this?

Reply:
This try catch was added when we had to ignore all the GRPC errors earlier when we were yet to add support for exactly once delivery. As we will be now propagating the error and take action based on it like retry or constructing AcknowledgementResult we no longer need to catch it and ignore here.